src/components/search/Search.jsx

```diff
@@ -34,15 +34,29 @@ const Search = () => {
         indexName={config.ALGOLIA_INDEX_NAME}
         searchClient={algolia.client}
       >
-        <Configure
-          hitsPerPage={NUM_RESULTS_PER_PAGE}
-          filters={filters}
-          clickAnalytics
-        />
         <div className="search-header-wrapper">
           <SearchHeader containerSize="lg" />
         </div>
-        <SearchResults />
+
+        <Index indexName={config.ALGOLIA_INDEX_NAME}>
+          <Configure
+            hitsPerPage={4}
+            filters={filters}
+            clickAnalytics
+          />
+          { /* A prop will be passed in SearchResults to specify if the search results are courses or programs */ }
+          { /* so it can decide which hit component to use SearchCourseCard or ProgramCard */ }
+          <SearchResults isCourses/>
+        </Index>
+
+        <Index indexName={config.ALGOLIA_INDEX_NAME}>
```

Comment:
It might also be good to wrap these `Index` components into their own `CourseSearch` and `ProgramSearch` top-level components.

Reply:
Done.